blocks/lead-art-block/themes/news.json

```diff
@@ -113,35 +51,40 @@
 			}
 		}
 	},
-	"lead-art-carousel-track-icon": {
+	"lead-art-track-button": {
 		"styles": {
 			"default": {
-				"fill": "var(--global-white)",
-				"height": "var(--global-spacing-8)",
-				"width": "var(--global-spacing-8)"
+				"color": "var(--global-white)",
+				"components": {
+					"button-hover": {
+						"color": "var(--global-neutral-3)"
+					},
+					"icon": {
+						"fill": "currentColor",
+						"height": "var(--global-spacing-8)",
+						"width": "var(--global-spacing-8)"
+					}
+				}
 			},
 			"desktop": {}
 		}
 	},
-	"lead-art-close-icon": {
+	"lead-art-close-button": {
```

Comment:
Same thing here, it's not matching what is in `_index.scss`

Reply:
Good catch! It is solved.
Thank you!